src/libImaging/ImPlatform.h

```diff
@@ -61,17 +83,22 @@
 #if SIZEOF_LONG == 8
 #define INT64 long
 #elif SIZEOF_LONG_LONG == 8
-#define INT64 long
+#define INT64 long long
+#else
+#warning Cannot find required 64-bit integer type
 #endif
 
-#define INT8 signed char
 #define UINT8 unsigned char
-
 #define UINT16 unsigned INT16
 #define UINT32 unsigned INT32
-
+#ifdef INT64
+#define UINT64 unsigned INT64
 #endif
 
+#endif /* C99+ */
+
+#endif /* WIN */
```

Comment:
And at least elsewhere the not case has been used for endif:

```suggestion
#endif /* < C99+ */

#endif /* not WIN */
```